packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx

```diff
@@ -54,16 +52,13 @@ export const BranchCard: React.FC<BranchProps> = ({
       <Stack
         css={css({
           backgroundColor: 'rgba(229, 229, 229, 0.04)',
-          paddingY: 10,
+          paddingY: 11,
           position: 'relative',
         })}
         align="center"
         justify="center"
       >
-        <Icon color="#808080" name="branch" size={40} />
-        <Element css={{ position: 'absolute', top: 6, right: 6 }}>
-          <CloudBetaBadge />
```

Comment:
We're not using the cloud beta badge anywhere anymore?

Reply:
I would still show it on the branch cards for the recent page, but we can merge this as it is as it fixes a lot of design insconsistencies and address it later?